designs.js

```diff
@@ -2,15 +2,21 @@ const listTitle = document.getElementById('inputTitle');
 const listQa= document.getElementById('inputQa');
 const listOfItems = document.getElementById('listBook');
 const submit = document.getElementById('MakeCatalog');
-  submit.addEventListener('submit', function (e){
-    e.preventDefault();
-    makeList();
-  }
-)
+const buttonAdd = document.getElementById("add");
+const saveButton = document.getElementById("saveButton");
+const clearButton = document.getElementById("clearButton");
+const inputDelete = document.getElementById("inputDelete");
+const up = document.getElementById("up");
```

Comment:
I would recommend to normalize naming of these constants. It helps when all names follow same pattern. Eg. "{action}{Element}" like "saveButton". Otherwise it makes reading code more difficult. The same is true for ids. In my view two lines below look great. All others should be changed in the same way.
```
const saveButton = document.getElementById("saveButton");
const clearButton = document.getElementById("clearButton");
```